testlib/testResources.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
```

Comment:
nit: I feel like we could name this file `resources.go` and avoid the stutter that is **test**lib.**test**Resources.go.